lib/core/macros.nim

```diff
@@ -339,11 +339,13 @@ proc quote*(bl: stmt, op = "``"): PNimrodNode {.magic: "QuoteAst", noSideEffect.
   ##       if not `ex`:
   ##         echo `info` & ": Check failed: " & `expString`
 
+from strutils import cmpIgnoreStyle, format
```

Comment:
The import was actually already in there ; I just moved it up.

There are other places that are using format in macros.nim as well,  so I will convert those to use string concatenation as well and remove the import completely.

On Mar 3, 2015, at 1:55 AM, Andreas Rumpf <notifications@github.com<mailto:notifications@github.com>> wrote:


In lib/core/macros.nim<https://github.com/Araq/Nim/pull/2248#discussion_r25674493>:

> @@ -339,11 +339,13 @@ proc quote*(bl: stmt, op = "``"): PNimrodNode {.magic: "QuoteAst", noSideEffect.
>    ##       if not `ex`:
>    ##         echo `info` & ": Check failed: " & `expString`
>
> +from strutils import cmpIgnoreStyle, format


A core module should not import anything. Use string concatenation instead of 'format' here.

—
Reply to this email directly or view it on GitHub<https://github.com/Araq/Nim/pull/2248/files#r25674493>.